pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -80,7 +80,7 @@ func NewMultiOrgAlertmanager(cfg *setting.Cfg, configStore AlertingStore, orgSto
 }
 
 func (moa *MultiOrgAlertmanager) setupClustering(cfg *setting.Cfg) error {
-	clusterLogger := moa.logger.New("component", "clustering")
+	clusterLogger := moa.logger.New("ngalert.notifier.multiorg_alertmanager")
```

Comment:
let's leave it as is because it just creates a wrapper for existing logger with additional context.

Reply:
Gotcha